app/controllers/users/passwords_controller.rb

```diff
@@ -81,9 +91,13 @@ def handle_expired_reset_password_token
       redirect_to new_user_password_path
     end
 
-    def handle_failed_password_reset_for(resource)
-      set_minimum_password_length
```

Comment:
Yep. All this Devise method does is create an instance variable that allows the view to display to the user the minimum required length for the password. We don't need that because the requirements are already hardcoded into the view.